site/en/docs/extensions/mv3/nativeMessaging/index.md

```diff
@@ -116,9 +116,14 @@ Sending and receiving messages to and from a native application is very similar
 messaging. The main difference is that [`runtime.connectNative`][connect-native] is used instead of
 [`runtime.connect`][connect], and [`runtime.sendNativeMessage`][send-native-message] is used instead of
 [`runtime.sendMessage`][send-message].
+
 To use these methods, the "nativeMessaging" permission must be [declared][declare-permissions] in your
 extensions's manifest file.
 
+These methods are not available inside content scripts, only inside your extension's pages and ServiceWorker. If you
+wish to communicate from a content script to the native application, send the message to your background script page
```

Comment:
Could we replace "ServiceWorker" with "service worker", and "background script page" with "background context"? I think that lines up nicer with our usual phrasing.

Reply:
done